setup.cfg

```diff
@@ -44,14 +44,19 @@ include_package_data = True
 [entry_points]
 tern.formats =
     default = tern.formats.default.generator:Default
-    spdxtagvalue = tern.formats.spdx.spdxtagvalue.generator:SpdxTagValue
-    spdxjson = tern.formats.spdx.spdxjson.generator:SpdxJSON
-    spdxjsonc = tern.formats.spdx.spdxjson.consumer:SpdxJSON
     json = tern.formats.json.generator:JSON
     jsonc = tern.formats.json.consumer:JSON
     yaml = tern.formats.yaml.generator:YAML
     html = tern.formats.html.generator:HTML
     cyclonedxjson = tern.formats.cyclonedx.cyclonedxjson.generator:CycloneDXJSON
+    spdxjson = tern.formats.spdx.spdxjson.generator:SpdxJSON
+    spdxyaml = tern.formats.spdx.spdxyaml.generator:SpdxYAML
+    spdxxml = tern.formats.spdx.spdxxml.generator:SpdxXML
+    spdxrdf = tern.formats.spdx.spdxrdf.generator:SpdxRDF
+    spdxtagvalue = tern.formats.spdx.spdxtagvalue.generator:SpdxTagValue
+    spdxtagvalue_legacy = tern.formats.spdx_legacy.spdxtagvalue.generator:SpdxTagValue
+    spdxjson_legacy = tern.formats.spdx_legacy.spdxjson.generator:SpdxJSON
+    spdxjsonc = tern.formats.spdx_legacy.spdxjson.consumer:SpdxJSON
```

Comment:
@armintaenzertng What's the idfference between `spdxjsonc` and `spdxjson`?

Reply:
`spdxjsonc` seems to provide the ability to parse SPDX files generated with tern and read from them the layer information. I have not touched that code. Seeing as you don't know about it either, is that feature still supported?
I'm trying to find out how to actually use it via the CLI.

Reply:
It seems like this is actually not meant to be called directly from the CLI, but instead is part of the `--live` option, where you can specify reports generated in previous runs via the `--with-context` flag.

I believe that both `jsonc` and `spdxjsonc` can therefore be removed from the entrypoints list, but seeing that I did not touch any of that code, I propose to do this in a separate PR.

Reply:
Oh ok, this was functionality Nisha worked on. I didn't realize it had its own entry point but let's leave it for now.